### ISSUE TITLE
/newrank/wechat/ is down

### ISSUE THREAD

**Issue by datatube-owner** [DIYgod/RSSHub#6439]
- The involved route, with all required and optional parameters

https://rsshub.app/newrank/wechat/chijiread

- What is expected

Normal content.

- What is actually happening

```
Route requested: /newrank/wechat/chijiread
Error message: Cannot read property 'name' of undefined
```

🔥

**Comment by paulhybryant** [comment]
+1, it is down

**Comment by TonyRL** [comment]
Closed in #12262
